setup.py

```diff
@@ -29,7 +29,7 @@
     install_requires=[
         "amazon-braket-schemas>=1.12.0",
         "amazon-braket-default-simulator>=1.10.0",
-        "oqpy==0.1.1",
+        "oqpy",
```

Comment:
Could we please change this to the compatible release clause instead? (`oqpy ~= 0.1.1`) This would ensure that we don't pull in an incompatible release without validating it first.

Reply:
Makes sense! Thanks for the reminder about this.